view/adminhtml/templates/catalog/product/attribute/options.phtml

```diff
@@ -0,0 +1,112 @@
+                <div data-role="draggable-handle" class="draggable-handle"
+                     title="<?= $block->escapeHtmlAttr(__('Sort Option')) ?>">
+                </div>
+            <?php endif; ?>
+            <input data-role="order" type="hidden" name="option[order][<%- data.id %>]"  value="<%- data.sort_order %>" <?php if ($block->getReadOnly() || $block->canManageOptionDefaultOnly()) :?> disabled="disabled"<?php endif; ?>/>
+        </td>
+        <td class="col-default control-table-actions-cell">
+            <input class="input-radio" type="<%- data.intype %>" name="default[]" value="<%- data.id %>" <%- data.checked %><?php if ($block->getReadOnly()) :?>disabled="disabled"<?php endif;?>/>
+        </td>
+        <td class="col-default">
+            <input class="input-text" type="text" name="option[code][<%- data.option_code %>]" value="<% if (data.option_code){ %><%-data.option_code %><% } %>" disabled/>
+        </td>
+        <?php foreach ($stores as $_store) :?>
+        <td class="col-<%- data.id %>">
+            <input name="option[value][<%- data.id %>][<?= (int) $_store->getId() ?>]" value="<%- data.store<?= /* @noEscape */ (int) $_store->getId() ?> %>" class="input-text<?php if ($_store->getId() == \Magento\Store\Model\Store::DEFAULT_STORE_ID) :?> required-option required-unique<?php endif; ?>" type="text" <?php if ($block->getReadOnly() || $block->canManageOptionDefaultOnly()) :?> disabled="disabled"<?php endif;?>/>
+            <?php endforeach; ?>
+        <td id="delete_button_container_<%- data.id %>" class="col-delete">
+            <input type="hidden" class="delete-flag" name="option[delete][<%- data.id %>]" value="" />
+            <?php if (!$block->getReadOnly() && !$block->canManageOptionDefaultOnly()) :?>
+                <button id="delete_button_<%- data.id %>" title="<?= $block->escapeHtmlAttr(__('Delete')) ?>" type="button"
+                        class="action- scalable delete delete-option"
+                >
+                    <span><?= $block->escapeHtml(__('Delete')) ?></span>
+                </button>
+            <?php endif;?>
+        </td>
+        </tr>
+    </script>
+    <?php
+    $values = [];
+    foreach ($block->getOptionValues() as $value) {
+        $value = $value->getData();
+        $values[] = is_array($value) ? array_map(function ($str) {
+            return htmlspecialchars_decode($str, ENT_QUOTES);
+        }, $value) : $value;
+    }
+    ?>
+    <script type="text/x-magento-init">
+        {
+            "*": {
+                "Magento_Catalog/js/options": {
+                    "attributesData": <?= /* @noEscape */ json_encode($values, JSON_HEX_QUOT) ?>,
+                    "isSortable":  <?= (int)(!$block->getReadOnly() && !$block->canManageOptionDefaultOnly()) ?>,
+                    "isReadOnly": <?= (int)$block->getReadOnly() ?>
+                },
+                "Magento_Catalog/catalog/product/attribute/unique-validate": {
```

Comment:
Maybe we need to look into how this works because I think at the moment if an option has the same admin value as another, you cannot save the page. Perhaps we should validate on option code uniqueness? What do you think @convenient ?